storage/innobase/handler/ha_innodb.cc

```diff
@@ -1104,6 +1104,10 @@ static SHOW_VAR innodb_status_variables[]= {
   {"encryption_num_key_requests", &export_vars.innodb_encryption_key_requests,
    SHOW_LONGLONG},
 
+  /* InnoDB bulk operations */
+  {"num_bulk_insert_operation",
+   &export_vars.innodb_bulk_insert_operation, SHOW_SIZE_T},
```

Comment:
The suggestion was `innodb_bulk_operations` in plural form. I don’t think that the components "num" or "insert" add much value.